pdc/apps/contact/models.py

```diff
@@ -8,11 +8,14 @@
 from django.db import models
 from django.db.models.query import QuerySet
 from django.forms.models import model_to_dict
+from django.core.exceptions import ValidationError
 
 
 class ContactRole(models.Model):
 
     name = models.CharField(max_length=128, unique=True)
+    count_limit = models.IntegerField(default=1)
```

Comment:
Adding a comment seems better that changing the name. Maybe [help text](https://docs.djangoproject.com/en/dev/ref/models/fields/#help-text) or just mention in docstring would be enough?